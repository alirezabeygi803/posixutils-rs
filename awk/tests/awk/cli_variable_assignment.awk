BEGIN {
	print variable;
	print _variable;
	print v2ar4_iable;
}