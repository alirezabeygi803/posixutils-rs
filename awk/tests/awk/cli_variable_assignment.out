value1
val
ue2
"value3"
